### ISSUE TITLE
[Bug]: Package Source options in VS does not announce package sources or checkbox status

### ISSUE THREAD

**Issue by zivkan** [NuGet/Home#11482]
### NuGet Product Used

Other/NA

### Product Version

17.0

### Worked before?

probably 16.7

### Impact

Other

### Repro Steps & Context

Open a screen reader of your choice. Windows Narrator is built into Windows, so doesn't require download or installation. Open VS, go to Tools->Options, NuGet Package Manager->Package Sources.  Now use either tab on the keyboard, or just click with the mouse. When it should be reading a package source from the list, it doesn't announce anything.

**Comment by akash07k** [comment]
@zivkan Thanks a lot for fixing this issue.
It's very helpful.
